backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxInitializerApi.scala

```diff
@@ -29,10 +29,16 @@ class VeloxInitializerApi extends InitializerApi {
   def loadLibFromJar(load: JniLibLoader): Unit = {
       val system = "cat /etc/os-release".!!
       if (system.contains("Ubuntu") && system.contains("20.04")) {
```

Comment:
val loader = if () {
new VeloxSharedlibraryLoaderUbuntu2004
}

loader.loadLib(load)

Reply:
Thanks, just addressed it.